home/templates/home/signup.html

```diff
@@ -0,0 +1,12 @@
+{% extends "home/base.html" %}
+
+{% block content %}
+<body class="main-bg">
+  <h2>Sign up</h2>
+  <form method="post">
+    {% csrf_token %}
+    {{ form.as_p }}
+    <button type="submit">Sign up</button>
```

Comment:
@idokk1 
I think you can link the "login_style.css"  file to the "signup.html" file.
Then:
1. wrap the sign-in form with div with `class= "content"`
2. add in the button tag `class="login-btn"`
so this page will be designed same as the login page.

Reply:
added css like the add new question page.